cluster/gce/gci/configure-helper.sh

```diff
@@ -675,14 +684,19 @@ contexts:
     user: kube-apiserver
   name: webhook
 EOF
-    # This is the config for the image review admission controller.
-    cat <<EOF >/etc/admission_controller.config
-imagePolicy:
+GCP_IMAGE_REVIEW_CONFIG="imagePolicy:
   kubeConfigFile: /etc/gcp_image_review.config
   allowTTL: 30
   denyTTL: 30
   retryBackoff: 500
   defaultAllow: true
+"
+  fi
+
+  if [[ -n "${GENERIC_ADMISSION_CONFIG:-}" || -n "${GCP_IMAGE_REVIEW_CONFIG:-}" ]]; then
```

Comment:
This is the same decision that line 1650 makes, also, right? I think if you make a `USING_ADMISSION_CONTROL_OPTIONS_FILE` variable, set it once, and then check it in both places, it will be more readable.

Reply:
set it to "true"?

Reply:
i just set it to 1
=